lib/rules/__tests__/index.test.mjs

```diff
@@ -32,16 +32,17 @@ describe('all rules', () => {
 	});
 });
 
-describe('fixable rules', () => {
-	test.each(ruleNames)('"%s" should describe fixable in the documents', async (name) => {
+describe('fixability', () => {
+	test.each(ruleNames)('"%s" should be properly documented', async (name) => {
 		const rule = await rules[name];
-
-		if (!rule.meta.fixable) return;
-
+		const cell = rule.meta.fixable ? '\\s+🔧\\s+' : ' ';
+		const rulesPattern = `^.+\`${name}\`.+\\|.+\\|${cell}\\|$`;
+		const needle = '`fix` option';
+		const rulePattern = rule.meta.fixable ? needle : `^((?!${needle})(\\s|\\S))*$`;
```

Comment:
[suggest] To be sure, is it not enough to use only `(?!pattern)` form like this?

```suggestion
		const rulePattern = rule.meta.fixable ? needle : `(?!${needle})`;
```

Reply:
I cannot test right now. Ill come back to you later.

Reply:
I just tried, it doesn't work.

Reply:
Okay, let's keep it.

Just curious, can you tell me the reason not to work?

Reply:
> Okay, let's keep it.

👍 

> Just curious, can you tell me the reason not to work?

I just put
```
`fix` option
```
in the README of `font-family-no-duplicate-names` and it doesn't fail the test with `(?!${needle})`.
Maybe there's something simpler that works too but I didn't bother.
Regular expressions are hard to get right :)

Reply:
I understand why, thanks 👍🏼 

> Regular expressions are hard to get right :)

Absolutely 😅